docs/workflows-config.md

```diff
@@ -58,6 +58,57 @@ Other points to note:
   problems caused by using conflicting versions of Bazel in different
   build environments.
 
+## Mac configuration
+
+By default, workflows will execute on BuildBuddy's shared Linux executors,
+but it is also possible to run workflows on macOS by using self-hosted
+executors.
+
+1. Set up one or more Mac executors that will be dedicated to running
+   workflows, following the steps in the [Enterprise
+   Mac RBE Setup](/docs/enterprise-mac-rbe) guide. Then, in your
+   `buildbuddy-executor.plist` file, find the `EnvironmentVariables`
+   section and set `MY_POOL` to `workflows`:
+
+```xml
+        ...
+        <key>EnvironmentVariables</key>
+        <dict>
+            ...
+            <key>MY_POOL</key>
+            <string>workflows</string>
+        </dict>
+        ...
+```
+
+2. If you haven't already, [enable workflows for your
+   repo](/docs/workflows-setup#enable-workflows-for-a-repo), then create a
+   file called `buildbuddy.yaml` at the root of your repo. See the
+   [Example config](#example-config) for a starting point.
+
+3. Set `os: "darwin"` on the workflow action that you would like to build
```

Comment:
nit: add an optional #4 to explain setting arch for m1 or old style?

Reply:
Done (kept it with step 3 to keep it short)